examples/timezone-aware.md

```diff
@@ -73,8 +73,8 @@ $('#clock').countdown(nextYear.toDate(), function(event) {
 
 ## Feature read
 
-- [MomentJS Documentation](http://momentjs.com/docs/)
-- [Moment Timezone Documentation](http://momentjs.com/timezone/docs/)
+- [MomentJS Documentation](https://momentjs.com/docs/)
```

Comment:
Not sure if we should assume that external links can handle secure connections. The same counts for all other links from the markdowns.